contrib/epee/src/byte_slice.cpp

```diff
@@ -133,10 +133,13 @@ namespace epee
 
   template<typename T>
   byte_slice::byte_slice(const adapt_buffer, T&& buffer)
-    : storage_(nullptr), portion_(to_byte_span(to_span(buffer)))
+    : storage_(nullptr)
```

Comment:
Explicit initialize `portion_(nullptr)`. Per C++ rules, its going to be initialized anyway to the same value, but it doesn't "look" possibly uninitialized that way.

Reply:
Added